python/lsst/ts/atmcssimulator/schemas/command_setInstrumentPort.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "id": {
+            "enum": [
```

Comment:
I think this is a "string" instead of "enum" (`"id": "cmd_setInstrumentPort"`). You would need to have the "cmd_" before each command to LabVIEW server. This applies to all the command related json file.